core/src/main/resources/hudson/model/Messages.properties

```diff
@@ -319,9 +319,9 @@ Permalink.LastFailedBuild=Last failed build
 Permalink.LastCompletedBuild=Last completed build
 
 ParameterAction.DisplayName=Parameters
-ParametersDefinitionProperty.DisplayName=This build is parameterized
+ParametersDefinitionProperty.DisplayName=This project is parameterized
```

Comment:
Not sure this shouldn't just say *Builds are parameterized* or *Builds have parameters*, because it's the builds that have the parameters.